app/adapters/widget.js

```diff
@@ -3,8 +3,8 @@ import JSONAPIAdapter from 'ember-data/adapters/json-api';
 import config from '../config/environment';
 
 export default JSONAPIAdapter.extend({
-
-    host: config.APP.GRANTS_BACKEND,
+    host: 'http://localhost:8000',
```

Comment:
Instead of defining the host in each adapter, you can define it once in the application adapter and extend it here and in the dashboard adapter